### PR TITLE
Updated README file

### PR DESCRIPTION
**Pull request by ai-fast-track**
Rewrite README file, added logo, and a self contained example with source code

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -44,25 +55,76 @@ Install pytorch via your preferred way.
...
+   valid_dl = model.dataloader(valid_ds, batch_size=8, num_workers=2)
+
+   # Use pre-trained backbone
+   resnet_101_backbone = MantisFasterRCNN.get_backbone_by_name("resnet101", fpn=True, pretrained=True)
+
+   # Create model
+   model = WheatModel(n_class=2, backbone=resnet_101_backbone)
+
+   # Train (fit) model
+   trainer = Trainer(max_epochs=2, gpus=1)
```
Comment (lgvaz) [inline README.rst:98]: I think the last line is missing, `trainer.fit(...)`...

`WheatModel ` is not defined, should we include it's code?

This example is also lightning specific, should we add this information?

I'm working on integration with fastai, should be ready tomorrow, it's something we can add as well =)

File: README.rst [hunk 2]
```diff
@@ -44,25 +55,76 @@ Install pytorch via your preferred way.
    pip install git+git://github.com/lgvaz/mantisshrimp.git
    pip install -U 'git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI'
 
-Quick start
------------
+Quick Example: How to train the **Wheat Dataset**
```
Comment (lgvaz) [inline README.rst:58]: Should we link to the wheat dataset kaggle page?

Or even better, we can use Pennfundan, I added functions for automatically downloading the dataset

```
from mantisshrimp.hub.pennfundan import *
source = get_pennfundan_data()
parser = PennFundanParser(source)
```

The problem is that the `PennFundanParser` is quite complicated.

File: README.rst [hunk 3]
```diff
@@ -8,7 +8,18 @@ MantisShrimp
 
 --------------
 
-``MantisShrimp`` provides a unified data API for object detection that can be used with any library (it can even be injected in code that was not supposed to work as a library, take a look at ``hub/detr`` for an example).
+.. image:: images/mantisshrimp-logo.png
+
+Why Mantishrimp
+---------------
+- Mantisshrimp: An object-detection library
+- Built on top of different libraries/framework such as Pytorch Lightning and Pytorch
```
Comment (lgvaz) [inline README.rst:16]: It's not really built on top of those libraries, rather it coexists with them. I'm not sure how to phrase it correctly, but the pluggable idea sounded good

File: README.rst [hunk 4]
```diff
@@ -8,7 +8,18 @@ MantisShrimp
...
+- Features a Unified Data API such: common Parsers (COCO, etc.),
+- User-defined Parsers (e.g. WheatParser)
```
Comment (lgvaz) [inline README.rst:18]: Can we say: "Community maintaned parsers for custom datasets"? Or something like that

File: README.rst (hunk 1 above)
Comment (ai-fast-track) [reply README.rst:98]: Ah! I forgot the fit! I was literally half sleeping. Sometimes, I hit the keys while I was snoozing!

Comment (ai-fast-track) [reply README.rst:98]: Great news wrt fastai!

File: README.rst (hunk 2 above)
Comment (ai-fast-track) [reply README.rst:58]: Maybe a toy dataset with COCO would be even better. The goal is to let the users to quick play with it in Colab.

File: README.rst (hunk 4 above)
Comment (ai-fast-track) [reply README.rst:18]: Maybe:
Integrates community maintaned parsers for custom datasets shared on mantisshrimp parsers hub

File: README.rst (hunk 3 above)
Comment (ai-fast-track) [reply README.rst:16]: We will rephrase it:

1st attempt:
Bridge different libraries/framework such as Pytorch Lightning and Pytorch

File: README.rst (hunk 2 above)
Comment (lgvaz) [reply README.rst:58]: I'll take a look at that balloon dataset I've been talking about, if I remember correctly it was COCO format

File: README.rst (hunk 3 above)
Comment (lgvaz) [reply README.rst:16]: You can put fastai in there ^^

File: README.rst (hunk 4 above)
Comment (lgvaz) [reply README.rst:18]: I like that!

Only suggestion: "mantisshrimp parsers hub" -> "parsers hub"

File: README.rst (hunk 3 above)
Comment (ai-fast-track) [reply README.rst:16]: 2nd attempt:
Connects to different libraries/framework such as Pytorch Lightning and Pytorch

Comment (lgvaz) [reply README.rst:16]: I think 2nd attempt is easier to understand

**Comment by oke-aditya** [comment]
Suggestion. Since Docker container stuff is already up. Can you add that as well?

File: README.rst (hunk 3 above)
Comment (ai-fast-track) [reply README.rst:16]: Connects to different libraries/framework such as fastai, Pytorch Lightning, and Pytorch

File: README.rst (hunk 2 above)
Comment (ai-fast-track) [reply README.rst:58]: Yes, let's stay far away from Facial Recognition!

**Comment by lgvaz** [comment]
@ai-fast-track I'm eager to merge this asap 😅

Are you still working on this or can I merge I make the proposed modifications?

**Pull request merged**